projects/batfish-common-protocol/src/org/batfish/datamodel/Configuration.java

```diff
@@ -72,6 +72,10 @@
 
    private static final String TACACS_SOURCE_INTERFACE_VAR = "tacacsSourceInterface";
 
+   private static final int VLAN_NORMAL_MIN_DEFAULT = 1;
```

Comment:
Are these constants common to all VLAN implementations for all vendors? ~If not, they should go in the respective VendorConfiguration classes in org.batfish.representation.XXX.XXXConfiguration.java~ It looks like you also have some constants in the VendorConfiguration classes. Why are there constants here?